include/sys/zap.h

```diff
@@ -307,7 +307,7 @@ int zap_join(objset_t *os, uint64_t fromobj, uint64_t intoobj, dmu_tx_t *tx);
 
 /* Same as zap_join, but set the values to 'value'. */
 int zap_join_key(objset_t *os, uint64_t fromobj, uint64_t intoobj,
-    uint64_t value, dmu_tx_t *tx, boolean_t exists_ok);
```

Comment:
This was just a cleanup of some changes that were left in here from the initial port. I don't think its bad to keep this function in the code. It could be useful at some point in the future and it would be nice to not have to rewrite it. I view it more as just an exported functionality of the ZAP code that nobody is using ATM.